### PR TITLE
mods to Java SDK doc to add publisher info

### PR DESCRIPTION
**Pull request by genwhittTTD**
mods to Java SDK doc to add publisher info

### CODE REVIEW

File: docs/sdks/uid2-sdk-ref-java.md [hunk 1]
```diff
@@ -102,6 +103,183 @@ Available information returned through the SDK is outlined in the following tabl
...
+
+2. Call a function that takes the user's email address or phone number as input and generates a `TokenGenerateResponse` object. The following example uses an email address:
+   ```java
+   TokenGenerateResponse tokenGenerateResponse = publisherUid2Client.generateTokenResponse(TokenGenerateInput.fromEmail(emailAddress).doNotGenerateTokensForOptedOut());
+   ```
+
+   :::important
+   <ul><li>Be sure to call this function only when you have obtained legal basis to convert the user’s [directly identifying information (DII)](../ref-info/glossary-uid.md#gl-dii) to UID2 tokens for targeted advertising.</li></ul>
+   
+   <ul><li>Always apply `doNotGenerateTokensForOptedOut()`. This applies `policy=1` in the [/token/generate](../endpoints/post-token-generate.md#token-generation-policy) call. Support for `policy=0` will be removed soon. (**GWH_SW01 to adjust this here + in the repo pending input. Readme line 53**)</li></ul>
```
Comment (sunnywu) [inline docs/sdks/uid2-sdk-ref-java.md:136]: _Support for `policy=0` will be removed soon._  can be omitted.

The rest can stay as is

File: docs/sdks/uid2-sdk-ref-java.md [hunk 2]
```diff
@@ -102,6 +103,183 @@ Available information returned through the SDK is outlined in the following tabl
...
+   :::
+
+#### Standard Integration
+
+If you're using standard integration (client and server) (see [UID2 SDK for JavaScript Integration Guide](../guides/publisher-client-side.md)), follow this step: (**GWH_SW02 not sure is this doc link correct, should we link to the client doc, or to both? Same in repo.**)
```
Comment (sunnywu) [inline docs/sdks/uid2-sdk-ref-java.md:141]: use this link

http://localhost:3006/docs/guides/integration-javascript-standard

File: docs/sdks/uid2-sdk-ref-java.md [hunk 3]
```diff
@@ -102,6 +103,183 @@ Available information returned through the SDK is outlined in the following tabl
...
+3. Using an HTTP client library of your choice, post this envelope to the [POST token/generate](../endpoints/post-token-generate.md) endpoint, including headers and body:
+   1. Headers: Depending on your HTTP library, this might look something like the following:  
+    
+      `.putHeader("Authorization", "Bearer " + UID2_API_KEY)`  
+      `.putHeader("X-UID2-Client-Version", PublisherUid2Helper.getVersionHeader())`
+   2. Body: `envelope.getEnvelope()`
+   :::important
+   - Be sure to call this endpoint only when you have obtained legal basis to convert the user’s [directly identifying information (DII)](../ref-info/glossary-uid.md#gl-dii) to UID2 tokens for targeted advertising.
+
+   - Always apply `doNotGenerateTokensForOptedOut()`. This applies `policy=1` in the [/token/generate](../endpoints/post-token-generate.md#token-generation-policy) call. Support for `policy=0` will be removed soon. (**GWH_SW03 same comment as earlier occurrence.**)
```
Comment (sunnywu) [inline docs/sdks/uid2-sdk-ref-java.md:215]: Again just remove
_Support for `policy=0` will be removed soon._

File: docs/sdks/uid2-sdk-ref-java.md [hunk 4]
```diff
@@ -102,6 +103,183 @@ Available information returned through the SDK is outlined in the following tabl
...
+
+4. If the HTTP response status code is _not_ 200, see [Response Status Codes](../endpoints/post-token-generate.md#response-status-codes) to determine next steps. Otherwise, convert the UID2 identity response content into a `TokenGenerateResponse` object:
+
+   ```java
+   TokenGenerateResponse tokenGenerateResponse = publisherUid2Helper.createTokenGenerateResponse({response body}, envelope);
+   ```
+
+#### Standard Integration
+
+If you're using standard integration (client and server) (see [UID2 SDK for JavaScript Integration Guide](../guides/publisher-client-side.md)): (**GWH_SW04 should this link to the client-side doc?**)
```
Comment (sunnywu) [inline docs/sdks/uid2-sdk-ref-java.md:226]: again it's the standard guide doc 
http://localhost:3006/docs/guides/integration-javascript-standard

File: docs/sdks/uid2-sdk-ref-java.md (hunk 1 above)
Comment (sunnywu) [reply docs/sdks/uid2-sdk-ref-java.md:136]: Maybe tweak this to:

_Always apply doNotGenerateTokensForOptedOut(). This applies a param similar to setting optout_check=1 in the [/token/generate](http://localhost:3006/docs/endpoints/post-token-generate#token-generation-policy) call._

policy=1 is not documented on the endpoint doc any more so we best not to say it. But i am trying to say doNOtGenerateTokensForOptedOut will apply policy=1 which has same effect as optout_check=1 so reader don't need to care about underlying implementation detail but just call the function. But maybe add something like:

_Not calling this doNOtGenerateTokensForOptedOut method is only available for early UID2 participants._ 

(and don't even mention policy=0 as they probably dunno what that is)

Applies this to comment further below too.

Comment (genwhittTTD) [reply docs/sdks/uid2-sdk-ref-java.md:136]: @sunnywu I fixed it to this (2 instances here, 2 in the repo readme):

IMPORTANT: Always apply doNotGenerateTokensForOptedOut(). This applies a parameter similar to setting optout_check=1 in the call to the POST /token/generate endpoint (see [Unencrypted JSON Body Parameters](https://unifiedid.com/docs/endpoints/post-token-generate#unencrypted-json-body-parameters)).

I'd prefer to leave out the other part about early UID2 participants. Would you be OK with that? If you feel strongly that we should include it let me know. Thx!

File: docs/sdks/uid2-sdk-ref-java.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/sdks/uid2-sdk-ref-java.md:141]: Done, two instances in this file and two instances in the repo readme, thx.

File: docs/sdks/uid2-sdk-ref-java.md (hunk 3 above)
Comment (genwhittTTD) [reply docs/sdks/uid2-sdk-ref-java.md:215]: Addressed same as your earlier comment, see query, thx.

File: docs/sdks/uid2-sdk-ref-java.md (hunk 4 above)
Comment (genwhittTTD) [reply docs/sdks/uid2-sdk-ref-java.md:226]: Done, two instances in this file and two instances in the repo readme, thx.

File: docs/sdks/uid2-sdk-ref-java.md (hunk 1 above)
Comment (sunnywu) [reply docs/sdks/uid2-sdk-ref-java.md]: I'm ok either way but if the early participants change something and they wanna update the vesion of sdk, they need to know to call doNotGenerateTokensForOptedOut() hence my rationale. Yeh maybe don't mention it for now.

**Review by sunnywu — APPROVED** [review]

File: docs/sdks/uid2-sdk-ref-java.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/sdks/uid2-sdk-ref-java.md]: @Sunny thank you. Merging.

**Pull request merged**